drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/factories/CollectionEditorSingletonDOMElementFactory.java

```diff
@@ -177,5 +177,10 @@ protected void commonCloseHandling(final CollectionEditorDOMElement collectionEd
         gridPanel.setFocus(true);
         collectionEditorDOMElement.stopEditingMode();
     }
+
+    @Override
+    protected CollectionEditorDOMElement createDomElementInternal() {
```

Comment:
@jomarko @manstis 
Please remove that. It depends on **status** of the instance (by itself, a bad practice) and, since it can't be _private_ for reusability, the only way to enforce/avoid potential NPE is to explicitily call with needed parameters

Reply:
Done in a separate commit: 'Make createDomElement undependent on class fields"